information-overload.md

```diff
@@ -0,0 +1,67 @@
+
+"I should really take that trip to the Bahamas"<br>
+"Now that I think about it, that grill of mine really doesn't work very well"<br>
+"Woah, the newest iPhone has such crisp picture quality!<br>
+"Those Balenciagas are soooooo in fashion, I need one!!
+
+It's consumerism at its highest peak, and when the brain is overloaded on information to the point where the body almost achieves a zombified state as it mindlessly scrolls, anyone can do or sell anything to you.
+
+Even to ignore the series of ads put within your face, the content itself also causes the brain to get overloaded with information, with user-generated content reaching an all-time peak in the history of the internet.[^Tang15trustinsocialmedia] YouTube reports that 500 hours of content are uploaded every minute. [^Fischbach20youtube] X reports that 350,000 tweets are sent every minute. [^Sayne22twitter] Facebook reports that 510,000 comments, 293,000 status updates, and 136,000 photos are uploaded every minute. [^Shepherd24facebook] Instagram reports that 95 million pieces of media are uploaded per day. [^Lister24instagram] There is so much user-generated content on these major platforms, and that doesn't even count TikTok, which does not show its amount of hours uploaded per minute, but can be assumed to be at least double that of YouTubes in the current year of 2024.
+
+If we were to take the following information:
+- 30,000 minutes of YouTube (500 * 60 minutes)
+- 90,000 minutes of TikTok (1,500 * 60 minutes)
+- 350,000 tweets
+- 510,000 comments, 293,000 status updates, 136,000 photos
+- 65,972 Instagram posts (95 million / 24 hours / 60 minutes)
+
+It adds up to 1,474,972 different pieces of information across 5 of the most popular social medias on the internet uploaded ***every single minute***.
+
+If we turn this number into meters, it would be equal to more than half the length of the Great Wall of China.
+
+What the brain has experienced while reading the numbers mentioned above may have been akin to information overload. The senses went blurry and the brain got fogged. Maybe the attention may have been diverted to somewhere else, or the brain may have shut off entirely as to save itself from physical pain by nothing but reading text. Information overload takes many forms, and the takeaway from any of what was mentioned above was this:
+
+## **Spend more time dealing with the problems in your head and mentally fortifying before information overload assault you all at once like a hurricane.**
+
+<br><br>
+
+[^Shepherd24facebook]:Shepherd, Jack. “33 Essential Facebook Statistics You Need to Know in 2024.” Social_Shepherd, February 26, 2024. https://thesocialshepherd.com/blog/facebook-statistics.
```

Comment:
Please double-check your citation style. You are not adhering to Chicago author-date.

Reply:
How exactly do I edit these changes? I can't figure out where to go